class.jetpack-admin.php

```diff
@@ -51,6 +52,34 @@ private function __construct() {
 
 		// Add module bulk actions handler
 		add_action( 'jetpack_unrecognized_action', array( $this, 'handle_unrecognized_action' ) );
+
+		// If the site has Jetpack Anti-Spam, change the Akismet menu label accordingly.
+		$site_products = Jetpack_Site_Products::get();
+		if ( false !== array_search( 'jetpack_anti_spam', array_column( $site_products, 'product_slug' ), true ) ) {
+			// Prevent Akismet from adding a menu item.
+			add_action(
+				'init',
+				function () {
+					remove_action( 'admin_menu', array( 'Akismet_Admin', 'admin_menu' ), 5 );
+				},
+				99
+			);
+			// Add an Anti-spam menu item for Jetpack.
+			add_action(
+				'jetpack_admin_menu',
+				function () {
+					add_submenu_page( 'jetpack', __( 'Anti-Spam', 'jetpack' ), __( 'Anti-Spam', 'jetpack' ), 'manage_options', 'akismet-key-config', array( 'Akismet_Admin', 'display_page' ) );
+				}
+			);
+
+			// This is a workaround to replace Akismet logo for Jetpacks without having to touch Akismet codebase. It
+			// should be removed once we have a Jetpack Anti-Spam setting page.
+			$logo            = new Jetpack_Logo();
+			$logo_base64     = base64_encode( $logo->get_jp_emblem_larger() );
+			$logo_base64_url = "data:image/svg+xml;base64,{$logo_base64}";
+			$style           = ".akismet-masthead__logo-container { background: url({$logo_base64_url}) no-repeat .25rem; height: 1.8125rem; } .akismet-masthead__logo { display: none; }";
+			wp_add_inline_style( 'admin-bar', $style );
```

Comment:
Getting an error in testing:

> Notice: wp_add_inline_style was called incorrectly. Scripts and styles should not be registered or enqueued until the wp_enqueue_scripts, admin_enqueue_scripts, or login_enqueue_scripts hooks. 

Can this be added that to the `admin_enqueue_scripts` hook?

Reply:
Thanks for catching this error. I fixed it using `admin_enqueue_scripts` as you suggested.